src/main/java/com/dabsquared/gitlabjenkins/connection/GitLabConnectionConfig.java

```diff
@@ -55,11 +55,11 @@ public boolean configure(StaplerRequest req, JSONObject json) throws FormExcepti
         return super.configure(req, json);
     }
 
-    public boolean isUseAuthenticatedEndpoint() {
+    public Boolean getUseAuthenticatedEndpoint() {
```

Comment:
I am 👎 as this causes binary incompatibility. This should be `isUseAuthenticatedEndpoint` as this is a normal java convention for boolean properties`is<propertyName>(){....}`